CONTRIBUTING.md

```diff
@@ -26,6 +29,13 @@ use GitHub pull requests for this purpose. Consult
 [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
 information on using pull requests.
 
+We require all commits in a PR to contain a `Signed-off-by` line which can be
+added by using the `-s` flag of `git commit`.
```

Comment:
We shouldn't simply tell folks to do this without telling them what it means:
https://developercertificate.org/
https://wiki.linuxfoundation.org/dco

Probably better to link out than summarize.

Reply:
Right. Added a link to the wiki.